### PR TITLE
fix #632

### LINKED ISSUE #632: container.mysql.lifecycle script should be copied from ConfMapVolumeMountPath to ConfVolumeMountPath
https://github.com/presslabs/mysql-operator/blob/b49149297775a21d1406a03fccdade37edde2a0d/pkg/controller/mysqlcluster/internal/syncer/config_map.go#L59

https://github.com/presslabs/mysql-operator/blob/b49149297775a21d1406a03fccdade37edde2a0d/pkg/sidecar/appconf.go#L38

I would fix it later on.

### LINKED ISSUE #637: mysqlbackup status can't be updated, there's always one running backup job
https://github.com/presslabs/mysql-operator/blob/5d86d0ce0c916dbadb70147e871d1f98ae80b18d/pkg/controller/mysqlbackup/mysqlbackup_controller.go#L182

`r.Update(context.TODO(), backup.Unwrap());` can't update mysqlbackup status, we need to update status of mysqlbackup separately. Demo as follows, is it right?

```
if !reflect.DeepEqual(savedBackup.Status, backup.Unwrap().Status) {
		log.Info("update backup object status")
		if err := r.Status().Update(context.TODO(), backup.Unwrap()); err != nil {
			log.Error(err, fmt.Sprintf("update status backup %s/%s", backup.Name, backup.Namespace),
				"backupStatus", backup.Status)
			return err
		}
}
```

### PR DESCRIPTION
**Pull request by jianhaiqing**
Fix #632  and fix #637

### CODE REVIEW

**Comment by jianhaiqing** [comment]
I can't open the error detail page, @AMecea  could you give me a hand ?

File: Changelog.md [hunk 1]
```diff
@@ -39,7 +39,8 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
  * Fix pod labels diff of map
  * Fixed backup cleanup job bug (#577)
  * Fix Kubebuilder path in Makefile.
-
+ * Fix #632
```
Comment (AMecea) [inline Changelog.md:42]: Please add a short description of the change.

**Comment by AMecea** [comment]
@jianhaiqing can I help you with the tests?

**Comment by jianhaiqing** [comment]
> @jianhaiqing can I help you with the tests?

I really appreciate it.

**Review by AMecea — APPROVED** [review]

**Pull request merged**